GameModes/Directory.Build.props

```diff
@@ -1,11 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)..\'))" />
-
-  <ItemGroup>
+  <ItemGroup Condition=" !$(MSBuildProjectName.EndsWith('.Tests')) ">
+    <PackageReference Include="Avalonia">
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
+    <PackageReference Include="ReactiveUI">
+      <ExcludeAssets>runtime</ExcludeAssets>
+    </PackageReference>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Core\Core.csproj">
```

Comment:
This is now referencing Core.csproj twice. I'm not really sure what that does.

Reply:
Whoops. Fixed both of these. :-)